sidebars.js

```diff
@@ -127,58 +134,75 @@ const sidebars = {
-            "tutorials/angular-todos/angular-todos-step-003",
-            "tutorials/angular-todos/angular-todos-step-004",
-            "tutorials/angular-todos/angular-todos-step-005",
-            "tutorials/angular-todos/angular-todos-step-006",
-          ],
+          type: "link",
+          label: "Developer Tutorials",
+          href: "https://amplication.com/tags/backend-development"
+        },
+        {
+          type: "link",
+          label: "Video Guides",
+          href: "https://www.youtube.com/@Amplicationcom/videos"
         },
         {
           type: "category",
-          label: "React Todos",
-          link: {
-            type: "doc",
-            id: "tutorials/react-todos/react-todos-step-000",
-          },
+          label: "Step-by-Step Tutorials",
           items: [
-            "tutorials/react-todos/react-todos-step-001",
-            "tutorials/react-todos/react-todos-step-002",
-            "tutorials/react-todos/react-todos-step-003",
-            "tutorials/react-todos/react-todos-step-004",
-            "tutorials/react-todos/react-todos-step-005",
-            "tutorials/react-todos/react-todos-step-006",
-          ],
-        },
-      ],
-    },
-    {
-      type: "category",
-      label: "Running Amplication Platform",
-      link: {
-        type: "doc",
-        id: "running-amplication-platform/connect-server-to-github",
-      },
-      items: [
-        "running-amplication-platform/connect-server-to-github",
-        "running-amplication-platform/configure-github-auth",
+            {
+              type: "category",
+              label: "Angular Todos",
+              link: {
+                type: "doc",
+                id: "tutorials/angular-todos/angular-todos-step-000",
+              },
+              items: [
+                "tutorials/angular-todos/angular-todos-step-001",
+                "tutorials/angular-todos/angular-todos-step-002",
+                "tutorials/angular-todos/angular-todos-step-003",
+                "tutorials/angular-todos/angular-todos-step-004",
+                "tutorials/angular-todos/angular-todos-step-005",
+                "tutorials/angular-todos/angular-todos-step-006",
+              ],
+            },
+            {
+              type: "category",
+              label: "React Todos",
+              link: {
+                type: "doc",
+                id: "tutorials/react-todos/react-todos-step-000",
+              },
+              items: [
+                "tutorials/react-todos/react-todos-step-001",
+                "tutorials/react-todos/react-todos-step-002",
+                "tutorials/react-todos/react-todos-step-003",
+                "tutorials/react-todos/react-todos-step-004",
+                "tutorials/react-todos/react-todos-step-005",
+                "tutorials/react-todos/react-todos-step-006",
+              ],
+            }
+          ]
+        }
       ],
     },
+    // {
```

Comment:
@GreenMachine01 @mulygottlieb: It's possible these pages under the Running Amplication Platform category are no longer relevant, but please let me know.